src/azure-cli/azure/cli/command_modules/monitor/_autoscale_util.py

```diff
@@ -662,3 +743,112 @@ def _profile_is_subset(profile, start, end):
             logger.warning("Proposed schedule '%s %s-%s' has a full or partial overlap with the following existing "
                            "schedules: %s. Unexpected behavior may occur.",
                            day, start, end, ', '.join(profile_conflicts))
+
+
+def build_autoscale_profile_dict(autoscale_settings):
```

Comment:
It is possible keep either `build_autoscale_profile_dict` or `build_autoscale_profile_from_instance`

Reply:
redesign after update migration done.